src/VisualizationTests/HelixImageComparisonTests.cs

```diff
@@ -10,18 +10,26 @@
 using System.Linq;
 using System.Reflection;
 using System.Windows.Media.Imaging;
-using Dynamo.Graph.Nodes.ZeroTouch;
+using Dynamo.Controls;
 using Dynamo.Selection;
+using Dynamo.Utilities;
 using DynamoCoreWpfTests.Utility;
 using HelixToolkit.Wpf.SharpDX;
 using NUnit.Framework;
 
 namespace WpfVisualizationTests
 {
-
     [TestFixture]
-    public class HelixImageComparisonTests : HelixWatch3DViewModelTests
+    public class HelixImageComparisonTests : VisualizationTest
```

Comment:
what does this look like before vs now in the test explorer in vs?

Reply:
Before:
<img width="708" alt="Screenshot 2023-07-10 at 3 06 54 PM" src="https://github.com/DynamoDS/Dynamo/assets/43763136/f7784c22-e813-4fff-b524-7a431db3cb10">

After:
<img width="709" alt="Screenshot 2023-07-10 at 3 10 41 PM" src="https://github.com/DynamoDS/Dynamo/assets/43763136/75f54584-25d0-402f-bbb9-6c43d57282c6">